woofbot-lnd/umbrel-app.yml

```diff
@@ -0,0 +1,33 @@
+manifestVersion: 1.1
+id: woofbot-lnd
+category: Finance
+name: WoofBot LND
+version: "0.7.0"
+tagline: A chatbot for your personal node
+description: >-
+  WoofBot is a chat bot that runs on your personal Bitcoin node and sends telegram notifications
+  based on pre-configured conditions: addresses, transactions, new-blocks and price changes.
+
+  This version of WoofBot integrates with LND lightning api. WoofBot is NOT affiliated with
+  Lightning Labs, the creators of LND. WoofBot only communicates with LND's api.
+developer: WoofBotApp
+website: https://github.com/woofbotapp
+dependencies:
+  - bitcoin
+  - lightning
+repo: https://github.com/woofbotapp/woofbotapp
+support: https://github.com/woofbotapp/woofbotapp/discussions
+port: 8093
```

Comment:
I doubled checked and this is a free port. Thanks!

Reply:
The original port for WoofBot (without Lightning) was 8092 - I think that you guys allocated it because my original pick was already taken. So 8093 for WoofBot-LND makes sense.